pipeline/transformer/transformer_common/src/main/scala/weco/pipeline/transformer/identifiers/IdentifierRegexes.scala

```diff
@@ -0,0 +1,63 @@
+package weco.pipeline.transformer.identifiers
+
+import scala.util.matching.Regex
+
+object IdentifierRegexes {
+  /*
+   * A Sierra system number (aka a bnumber) is composed of:
+   * - A prefix 'b'
+   * - A 7 digit identifier
+   * - A check character, which can be any digit or an 'x'
+   */
+  lazy val sierraSystemNumber: Regex = "^b[0-9]{7}[0-9x]$".r
```

Comment:
Depending on how strict you want to be, you could also bring in the SierraRecordNumber class, which validates a correct check digit – but that's probably unnecessary.

Reply:
Yeah, think I'll leave it here